src/nopoll_helpers.c

```diff
@@ -57,9 +66,14 @@ void sendMessage(noPollConn *conn, void *msg, size_t len)
     static int connErr=0;
     long timeDiff = 0;
 
+    if (!cloud_status_is_online ()) {
+		ParodusError ("Cloud Status not ONLINE at nopoll send\n");
+		return;
+	}
+
     ParodusInfo("sendMessage length %zu\n", len);
 
-    if(nopoll_conn_is_ok(conn) && nopoll_conn_is_ready(conn))
+    if(nopoll_conn_is_ok(conn) && conn->handshake_ok)
```

Comment:
I guess we should remove this entire nopoll conn check? I dont think its good to use conn internal variable here in parodus.

Reply:
I still think we need this check.

So I propose that in nopoll we create two functions:
  nopoll_conn_make_ready (which is the same as current nopoll_conn_is_ready), but this is a better name, anyway.
  nopoll_conn_is_ready (which just checks the handshake, but does not complete it)

This means we have to fix all the current calls to nopoll_conn_is_ready, but that is easy.

In parodus there is one call to nopoll_conn_is_ready (in sendMessage), which will now call the newly defined function.

In nopoll, I found four (4) calls of nopoll_conn_is_ready, all in nopoll_conn.c, and these must be changed to nopoll_conn_make_ready.

Reply:
OK. I get it.  We don't need this check.